tequilapi/endpoints/sessions.go

```diff
@@ -111,7 +112,7 @@ func sessionToDto(se session.Session) SessionDTO {
 		ProviderID:      string(se.ProviderID.Address),
 		ServiceType:     se.ServiceType,
 		ProviderCountry: se.ProviderCountry,
-		DateStarted:     se.Started.Format("2018-10-29 16:22:05"),
+		DateStarted:     se.Started.UTC().Format(time.RFC3339),
```

Comment:
I think it would be better to set time in UTC before persisting session itself, not when rendering in presentation layer. I assume RFC3339 formatting includes timezone too?

Reply:
It does - any reason why you don't want the UTC time there?

Reply:
Start time in UTC must be set on session creation - my point is, let's keep times in UTC in database too.